crates/biome_js_analyze/src/lint/nursery/no_restricted_imports.rs

```diff
@@ -36,14 +41,24 @@ declare_rule! {
     }
 }
 
+/// Options for each path of rule `noRestrictedImports`.
+#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct RestrictedModuleConfig {
+    pub message: String,
+    pub allowed_from: Option<Vec<String>>,
+    pub include_all_submodules: Option<bool>,
```

Comment:
There is no need for `Option` here; a `bool` should be fine, I think.

Reply:
Here is the same issues as the Option on the Vec. Without it the warning appears ("Missing property 'include_all_submodules'"). But if this is not a problem, I can removed them both.